commands/banner.rs

```diff
@@ -0,0 +1,69 @@
+use poise::serenity_prelude::{AttachmentType, ChannelId, CreateEmbed, EmbedMessageBuilding};
+
+/// Command to submit an image for the banner rotation
+#[poise::command(
+    prefix_command,
+    slash_command,
+    guild_only,
+    required_bot_permissions = "SEND_MESSAGES",
+)]
+pub async fn submit_image(
+    ctx: Context<'_>,
+    #[description = "Image to submit"] image: AttachmentType,
+) -> Result<(), Error> {
+    let guild_id = ctx.guild_id().ok_or(CommandErr::GuildOnly)?;
+
+    // Define the private channel ID where submissions will be sent
+    let private_channel_id = ChannelId(123456789012345678); // Replace with your private channel ID
```

Comment:
This bot is designed to run in multiple guilds so a hard coded value at this point would not work :<

Reply:
Ah yes forgot about that, I'll work on changing it